rally/.gitignore

```diff
@@ -0,0 +1,4 @@
+documents/*.json.offset
```

Comment:
When Rally downloads data from a remote server, it will end up in `~/.rally/benchmarks/data/apm-events` instead of the current directory (it is just that Rally will search both the current directory and the data directory I mentioned before for convenience when you develop locally).